### PR TITLE
fix(seo): Provide Organisation structured data

### PR DESCRIPTION
**Pull request by shawn-dsz**
In order to help google better understand our pages we need to provide structured data via script tag.

This PR adds a `<script />` tag inside the `<Header />` to add the structured organization schema.

Ref:
[Introduction to Structured Data](https://developers.google.com/search/docs/guides/intro-structured-data)

### CODE REVIEW

File: react/Header/Header.test.js [hunk 1]
```diff
@@ -45,4 +45,8 @@ describe('Header:', () => {
   it('should append returnUrl to signin and register links if present', () => {
     expect(renderHeader({ returnUrl: '/jobs' })).toMatchSnapshot();
   });
+
+  it('should append custom schema', () => {
+    expect(renderHeader({ structuredDataSchema: { org: 'SEEK' } })).toMatchSnapshot();
```
Comment (markdalgleish) [inline react/Header/Header.test.js]: Is the `structuredDataSchema` prop meant to be here? It doesn't look like the `Header` component does anything with this, is that right?

File: react/Header/StructuredDataSchema/generate-structured-data-schema.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+const generateOrgSchema = (locale = 'AU') => {
+  const { telephone, socialLinks, url } = locale === 'NZ' ? nzOrgSchema : auOrgSchema;
+
+  return {
+    '@context': 'http://schema.org',
+    '@type': 'Organization',
+    url,
+    legalName: 'Seek Limited',
+    logo: 'https://www.seek.com.au/content/images/logos/seek-logo-positive.svg',
```
Comment (markdalgleish) [inline react/Header/StructuredDataSchema/generate-structured-data-schema.js]: Do we know who officially owns this URL, how it's managed, what the support policy is, etc? Might need to take this conversation offline.

File: react/Header/Header.test.js (hunk 1 above)
Comment (shawn-dsz) [reply react/Header/Header.test.js]: oops, this needs to be gone!

**Comment by markdalgleish** [comment]
What happens when you need to provide more structured data beyond the defaults? Does it render additional script tags? If so, is that okay?

**Comment by shawn-dsz** [comment]
If you need to need to provide additional schema data; all you need to do is create another script tag on the page to compliment this. Having multiple script tags on the page is allowed. The idea here is that each seek app would include this common meta & supplement page specific meta in the consuming apps.

File: react/Header/StructuredDataSchema/generate-structured-data-schema.test.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        '@type': 'ContactPoint',
+        telephone: '+61-1300-658-700',
+        contactType: 'customer service',
+        contactOption: 'TollFree'
+      }
+    ]
+  });
+});
+
+test('should generate default (AU) organisation schema', () => {
```
Comment (michaeltaranto) [inline react/Header/StructuredDataSchema/generate-structured-data-schema.test.js]: Typo, `should generate *NZ* organisation schema`

File: react/Header/StructuredDataSchema/generate-structured-data-schema.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  ],
+  telephone: '+64-0508-733-569',
+  url: 'https://www.seek.co.nz/'
+};
+
+const commonSocialLinks = [
+  'https://www.facebook.com/SEEK/',
+  'https://en.wikipedia.org/wiki/Seek_Limited',
+  'https://www.youtube.com/user/SEEKJobs',
+  'https://au.linkedin.com/company/seek'
```
Comment (kosanna) [inline react/Header/StructuredDataSchema/generate-structured-data-schema.js]: Isn't linkedin separate? https://nz.linkedin.com/company/seek

**Comment by kosanna** [comment]
Is that okay that the schema is high-level and does not have anything page-specific?

**Comment by shawn-dsz** [comment]
@kosanna That's the idea, this PR is to  high-level  organization schema that will applied to all of SEEK's pages. On each we can then add page specific schema, as mentioned [above](https://github.com/seek-oss/seek-style-guide/pull/309#issuecomment-327638386).

**Comment by kosanna** [comment]
Ran  them through validator, looks good 👍

File: react/Header/StructuredDataSchema/generate-structured-data-schema.js (hunk 4 above)
Comment (shawn-dsz) [reply react/Header/StructuredDataSchema/generate-structured-data-schema.js]: Good pick-up, yup :slightly_smiling_face:

**Pull request merged**